paper.tex

```diff
@@ -108,12 +108,12 @@ \subsubsection{Backward Jump}
     goto
       [g]
         seq > @
-          i.write (i.plus 1)
+          i.lt (i.plus 1)
```

Comment:
@Graur this sounds like a typo. are you sure?

Reply:
@yegor256 Indeed, it's a typo